tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "sourceMap": false,
     "resolveJsonModule": true,
     "types": ["@types/node"],
+    "strict": false,
```

Comment:
Why can't we use strict:true ?

Reply:
I am currently running the CI with strict: true. I did this experiment because of failing CI's in ACI. After this last run we can merge this version and we can remove the install of rollup from privacy-sandbox-dev